index.js

```diff
@@ -0,0 +1,13 @@
+function isJSON(str) {
+    try {
+        JSON.parse(str);
+        console.log(true);
```

Comment:
Will be better to return value, you can use the returned value in other functions